conf/base.config

```diff
@@ -2,63 +2,20 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     sanger-tol/sequencecomposition Nextflow base config file
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    A 'blank slate' config file, appropriate for general use on most high performance
-    compute environments. Assumes that all software is installed and available on
-    the PATH. Runs in `local` mode - all jobs will be run on the logged in environment.
-----------------------------------------------------------------------------------------
 */
 
 process {
 
-    // TODO nf-core: Check the defaults for all processes
-    cpus   = { check_max( 1    * task.attempt, 'cpus'   ) }
-    memory = { check_max( 6.GB * task.attempt, 'memory' ) }
-    time   = { check_max( 4.h  * task.attempt, 'time'   ) }
-
     errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
-    maxRetries    = 1
+    maxRetries    = 2
```

Comment:
Thinking further about that, `sanger.config` shouldn't override `maxRetries` or `errorStrategy` because these can be pipeline-specific. In which order are those config files loaded ?

Reply:
Configs are loaded in the following order, the later ones override the earlier ones:
1. `nextflow.config`
2. `conf/base.config`
3. `nfcore_custom.config`
4. `conf/test.config`
5. `conf/test_full.config`
6. `conf/igenomes.config`
7. `conf/modules.config`
8. custom configs